README.md

```diff
@@ -348,6 +348,11 @@ req.session.reload(function(err) {
 })
 ```
 
+#### Session.load(sid, callback)
```

Comment:
I don't think this is part of this module, though. Can you link to where it is in source? I see no load method on our prototype, unless I am just looking past it.

Reply:
Thanks. Hmm, I wonder why it is not documented... We should get it documented as a separate PR (and add tests if those are missing too), not bundled into this one.

Reply:
Yea, I didn't say there were no tests, only that documenting it (and adding tests _if those are missing too_) just needs to be moved into a separate PR.

Reply:
Oh, looking at that test, we indeed do not have `store.load` tests. It would be around the _public usage_ of that method (which is what documenting it in the README is for) and would be similar in scope to the other documented methods like https://github.com/expressjs/session/blob/3b08fc7a78fe7742d5a29c5a4ad999169adaf3cb/test/session.js#L1611 or https://github.com/expressjs/session/blob/3b08fc7a78fe7742d5a29c5a4ad999169adaf3cb/test/session.js#L1570 or https://github.com/expressjs/session/blob/3b08fc7a78fe7742d5a29c5a4ad999169adaf3cb/test/session.js#L1725 etc.

But regardless of the above, I do wonder: why would a store who is opting for support req passing not then take the req as an argument to this load method like it does for all the others?

Reply:
> https://github.com/expressjs/session/blob/master/session/store.js#L67

P.S. That would be documented as `Store.load`, not `Session.load`. That is why I didn't see it went it went to look, as it's a method on the `Store` class and not the `Session` class.